package.json

```diff
@@ -7,27 +7,34 @@
     "test": "mocha $(find . -name '*.spec.js' ! -ipath '*node_modules*')",
     "build": "npm run build:src && npm run build:bundle",
     "build:src": "webpack --optimize-minimize",
-    "build:bundle": "BUNDLE=true webpack --optimize-minimize"
+    "build:bundle": "BUNDLE=true webpack --optimize-minimize",
+    "eslint": "eslint **.js",
+    "eslint:fix": "eslint --fix **.js"
```

Comment:
These commands won't reach directories, e.g. `example`. (And there are lint errors in there.)